src/GriffendorFaculty.java

```diff
@@ -0,0 +1,35 @@
+public class GriffendorFaculty {
+
+
+
+
+
+    private String Name;
```

Comment:
Имена свойств с маленькой буквы
```suggestion
    private String name;
```